### PR TITLE
Node ssl docs

### PR DESCRIPTION
**Pull request by cstrangedk**

### CODE REVIEW

File: validators/register-validator-node.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ SKALE Validator CLI is the validator client interface for registering a new vali
 -   Link all validator node addresses to a validator wallet address
 -   Request or cancel a delegation
 
-See the SKALE Validator CLI code and documentation on [**GitHub**](https://github.com/skalenetwork/validator-cli)*‍
+See the SKALE Validator CLI code and documentation on [**GitHub**](https://github.com/skalenetwork/validator-cli)\*‍
```
Comment (ebruengwall-ant) [inline validators/register-validator-node.md]: What is the change here?

File: validators/register-validator-node.md [hunk 2]
```diff
@@ -355,19 +373,20 @@ sudo docker-compose stop
...
 ### **Prerequisites**
 
 -   A Linux x86_64 machine
 -   SGX-enabled Intel processor
--   Ports 22, 8080, 9100, and 10000-11000, and ICMP IPv4 open for all
+-   Ports 22, 8080, 9100, and 10000–11000, and ICMP IPv4 open for all
+-   SSL Certificates
 -   Ubuntu 18.04 or later LTS
--   2TB attached storage main-net (200gb devnet)
+-   2TB attached storage main-net (200gb Testnet)
```
Comment (ebruengwall-ant) [inline validators/register-validator-node.md]: can we also add notification here (unmounted) We had issues with mounted disks during initialization

File: validators/register-validator-node.md [hunk 3]
```diff
@@ -690,89 +710,99 @@ After the previous step delegation status will be seen as accepted.
...
+Verify that redirects work by sending a request to the watchdog API on 3009 port using your domain name.
 
-```bash
-skale node register --name [NODE_NAME] --ip [NODE_IP] --port [PORT]
+### Step 4.3: Issue SSL certificates
 
-```
-
-Optional arguments:
+You will need SSL certs issued by one of the Trusted CAs. Once you've decided on the certificate issuer you have several options - issue a separate certificate for each subdomain (node-0.awesome-validator.com, node-1.awesome-validator.com) or issue a single Wildcard SSL for all nodes (\*.awesome-validator.com). As a result, you should have 2 main files saved and copied to the respective nodes:
```
Comment (ebruengwall-ant) [inline validators/register-validator-node.md:727]: Why 2?

Comment (ebruengwall-ant) [reply validators/register-validator-node.md:727]: you mean the certificate and private key file?

File: validators/register-validator-node.md [hunk 4]
```diff
@@ -690,89 +710,99 @@ After the previous step delegation status will be seen as accepted.
...
 </Step>
 
 <Step id='five'>
 
-## 5: Upload new SSL certificates
+## 5: Register Node in SKALE Network
+
+### Step 5.1: Register Node with Node CLI
+
+Note: Before proceeding, you will need to have at least  **0.2 Test ETH**. Also amount of delegated skale tokens need to be more or equal to minimum staking amount. Otherwise you won't be able to register with the SKALE Testnet.
```
Comment (ebruengwall-ant) [inline validators/register-validator-node.md]: Validators are using this walk through also for Mainnet. Maybe add `For testnet? or **0.2 ETH**

File: validators/register-validator-node.md [hunk 5]
```diff
@@ -690,89 +710,99 @@ After the previous step delegation status will be seen as accepted.
...
-Admin API URL: \[GET] `/api/ssl/status`
+**Output:**
+
+```shell
+# Node info
+Name: $NODE_NAME
+IP: $NODE_IP
+Public IP: <Public IP of Machine>
+Port: $NODE_PORT
+Status: Active
```
Comment (ebruengwall-ant) [inline validators/register-validator-node.md:806]: No domain output?

File: validators/register-validator-node.md [hunk 6]
```diff
@@ -690,89 +710,99 @@ After the previous step delegation status will be seen as accepted.
...
+2.  Machine public IP
+3.  Port - beginning of the port range that will be used for skale schains (10000 recommended)
+4.  Domain name
 
 **Terminal Command:**
 
-```bash
-skale ssl upload
+```shell
+skale node register --name [NODE_NAME] --ip [NODE_IP] --port [PORT] --domain [DOMAIN_NAME]
```
Comment (ebruengwall-ant) [inline validators/register-validator-node.md:774]: Is domain optional or required in the command?

File: validators/node-ssl.adoc [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+:warning-caption: :warning:
+
+endif::[]
+
+toc::[]
+
+== General info
+Node SSL certificates support secure communication with sChain endpoints. By default, each node of a SKALE Chain listens on HTTP and WS ports. If SSL certs exist on the node, then HTTPS and WSS ports are also turned on. 
+
+Node SSL certificates aren't required to start a node, register it, or create sChains. However, dApp developers using SKALE Chains need HTTPS/WSS endpoints to work with toolings such as API based wallets and other integrations. Therefore, SSL certificates are required for SKALE Network validator nodes.
```
Comment (ebruengwall-ant) [inline validators/node-ssl.adoc:20]: Does it make sense to mention that features like IMA require HTTPS as well?

File: validators/node-ssl.adoc [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+
+=== Setting domain name of a registered node
+
+To set the domain name of the registered node with SKALE manager, execute the following node-cli command on each respective node:
+
+```shell
+skale node set-domain -d node-0.awesome-validator.com
+```
+
+=== Registering domain name for a new node
```
Comment (ebruengwall-ant) [inline validators/node-ssl.adoc]: can we change to `Registering a new node with a domain name `

File: validators/register-validator-node.md (hunk 3 above)
Comment (dmytrotkk) [reply validators/register-validator-node.md:727]: @ebruoguzberk yes, certificate and key

File: validators/register-validator-node.md (hunk 5 above)
Comment (dmytrotkk) [reply validators/register-validator-node.md:806]: @ebruoguzberk actually we've added domain name and node id to the `skale node info` cmd

File: validators/register-validator-node.md (hunk 6 above)
Comment (dmytrotkk) [reply validators/register-validator-node.md:774]: @ebruoguzberk required

File: validators/register-validator-node.md (hunk 5 above)
Comment (dmytrotkk) [reply validators/register-validator-node.md:806]: updated

File: validators/node-ssl.adoc (hunk 8 above)
Comment (cstrangedk) [reply validators/node-ssl.adoc]: changed doc layout

File: validators/register-validator-node.md (hunk 4 above)
Comment (cstrangedk) [reply validators/register-validator-node.md]: fixed

File: validators/register-validator-node.md (hunk 2 above)
Comment (cstrangedk) [reply validators/register-validator-node.md]: added unmounted note

File: validators/register-validator-node.md (hunk 1 above)
Comment (cstrangedk) [reply validators/register-validator-node.md]: deleted

**Review by gannakulikova — DISMISSED** [review]

File: validators/node-ssl.adoc (hunk 7 above)
Comment (ebruengwall-ant) [reply validators/node-ssl.adoc:20]: It's not required to use the IMA. It's just required from the customers. So resolving this comment

**Review by ebruengwall-ant — DISMISSED** [review]

**Review by ebruengwall-ant — APPROVED** [review]

**Pull request merged**